osu.Game/Overlays/Settings/Sections/Maintenance/GeneralSettings.cs

```diff
@@ -29,17 +30,17 @@ public class GeneralSettings : SettingsSubsection
         private TriangleButton undeleteButton;
 
         [BackgroundDependencyLoader(permitNulls: true)]
-        private void load(BeatmapManager beatmaps, ScoreManager scores, SkinManager skins, [CanBeNull] CollectionManager collectionManager, DialogOverlay dialogOverlay)
+        private void load(BeatmapManager beatmaps, ScoreManager scores, SkinManager skins, [CanBeNull] CollectionManager collectionManager, [CanBeNull] StableImportManager stableImportManager, DialogOverlay dialogOverlay)
         {
-            if (beatmaps.SupportsImportFromStable)
+            if (stableImportManager?.SupportsImportFromStable ?? false)
```

Comment:
Another code style thing but I've really grown to dislike that `if (... ?? false)` sort of pattern and prefer `if (... == true)`. It has less of that weird double negation logic (`?? false` on nullable bools feels like writing `!= false` for normal bools).

If you do decide that you want to fix there's more usages in this diff that  I'm going to skip over for now.

Reply:
Strongly agree with this. If possible we should add an inspection/rule to prefer this style.